spearhead/assets/sass/_text.scss

```diff
@@ -1,19 +1,18 @@
-.has-link-color a, a {
-    border-bottom: 1px solid transparent;
+.has-link-color, a {
```

Comment:
I think this should be `.has-link-color a, a`, otherwise when you add a link color setting it does this:

<img width="735" alt="Screenshot 2021-01-27 at 17 06 35" src="https://user-images.githubusercontent.com/275961/106026745-1b20fa00-60c2-11eb-84eb-09e8985e6019.png">

Reply:
Ah, now I see.  Thank you.